chef/cookbooks/rabbitmq/recipes/ha.rb

```diff
@@ -332,6 +390,18 @@
     action [:stop, :delete]
     only_if "crm configure show #{public_vip_primitive}"
   end
+
+  pacemaker_primitive public_portblock_primitive do
```

Comment:
I was wondering about whether the ordering of the stop+delete pairs matters here, but I can't think of a convincing argument why it would, so this looks OK.